custom_components/myskoda/switch.py

```diff
@@ -496,3 +501,113 @@ async def async_turn_on(self, **kwargs):  # noqa: D102
 
     def required_capabilities(self) -> list[CapabilityId]:
         return [CapabilityId.AIR_CONDITIONING_SMART_SETTINGS]
+
+
+class DepartureTimerSwitch(MySkodaSwitch):
+    """Base class for departure timers, handling common functionality."""
+
+    def __init__(self, coordinator, vin, timer_id: int, **kwargs):
+        """Initialize the departure timer switch."""
+        super().__init__(coordinator, vin)  # Initialize parent class (MySkodaEntity)
+        self.timer_id = timer_id  # Store the specific timer ID for each subclass
+
+    def get_timer(self) -> Optional[DepartureTimer]:
```

Comment:
While this is valid, we have multiple points in the code where we use `State | None` instead of `Optional[] `.  Please use `State | None` for consistency sake.